frontend/src/Commons/Forms/InputModules/CalculateNutrients/CalculateNutrientsList.tsx

```diff
@@ -84,7 +84,7 @@ const CalculationList: FC<CalculationListProps> = ({ field, cropBalances, result
                   <td>
                     <h4 className="col1">Fertilizer</h4>
                   </td>
-                  <td>
+                  <td aria-hidden="true">
```

Comment:
@GDamaso this was an eslint error that I was having that was getting blocked by husky. Just in case you're wondering.

Reply:
Weird that it didn't show up for me